examples/core-tutorial/app/layout.tsx

```diff
@@ -9,19 +10,21 @@ const NAVIGATION: Navigation = [
     title: 'Main items',
   },
   {
-    segment: '/page',
+    segment: 'page',
     title: 'Page',
     icon: <DashboardIcon />,
   },
 ];
 
 export default function RootLayout({ children }: Readonly<{ children: React.ReactNode }>) {
   return (
-    <html lang="en">
+    <html lang="en" data-toolpad-color-scheme="light">
```

Comment:
Matching the playground should be good here!

Reply:
I think the playground app also contains the same attribute in the `app/layout.tsx` file, isn'it? That's what I can see here: https://github.com/mui/mui-toolpad/blob/master/playground/nextjs/src/app/layout.tsx#L27

Reply:
Yep that's what I meant, was just confirming that it seems good!